document/core/syntax/values.rst

```diff
@@ -145,6 +145,19 @@ Conventions
 
 * The meta variable :math:`z` ranges over floating-point values where clear from context.
 
+.. index:: ! simd, fixed-width, vector
+   pair: abstract syntax; packed simd number
+.. _syntax-simd:
+
+Fixed-Width SIMD
+~~~~~~~~~~~~~~~~
+
+Fixed-width SIMD data represents packed integers, or floating-point values. The packed values of a given SIMD value must all be of the same type, which can be any of the signed, unsigned integer types, and floating-point types ocurring in this proposal. The supported representations of the packed values are 4 32-bit, or 2 64-bit |IEEE754|_ numbers, or different widths of packed integer values specifically 2 64-bit integers, 4 32-bit integers, 8 16-bit integers, or 16 8-bit integers.
```

Comment:
The abstract syntax needs to have one unique way of representing each SIMD value. Reinterpretation has to be expressed by explicit meta functions, similar to the "signed" function for integers.

The most canonical choice for a representation would simply be as a 128-bit integer, i.e. the spec just uses `i128` for SIMD values. Then this section doesn't need to define anything, except maybe include a note saying that i128 is also used to represent SIMD values.

At some place (probably along with the SIMD ops) the spec then has to define interpretation functions like `i16x8` that convert this to a sequence i16^8 and so on. For example,
```
i64x2(i) = i_1 i_2
where
  b* = bytes_i128(i)
  i_1 = bytes_i64^{-1}(b*[0 \slice 4])
  i_2 = bytes_i64^{-1}(b*[4 \slice 4])
```

The definition of iN above is generic, so already defines i8, i16, i128. Likewise, all the ops in the numerics section are generic.